### ISSUE TITLE
Lock Rambox doesn't work on second try after entering incorrect password on first try

### ISSUE THREAD

**Issue by brendanjones** [ramboxapp/community-edition#1011]
### Steps to reproduce

1. Click to lock Rambox. Enter a password, then a different password when you are asked to repeat it
2. Enter the password, entering the same password twice (correctly) this time
3. Rambox doesn't lock :'(

### Expected behavior

Should lock!

### Actual behavior

Returns to normal operating environment

### ENVIRONMENT

> Rambox: latest

![screen shot 2017-07-21 at 11 59 42](https://user-images.githubusercontent.com/16049594/28460207-5349cb2a-6e11-11e7-93dd-3d92bcf1df46.png)

**Comment by brendanjones** [comment]
Also, if you do the above (enter non-matching passwords on first try, then matching on second try - Rambox doesn't lock), but then Reload Rambox, Rambox reloads as locked.

**Comment by saenzramiro** [comment]
Do you still having this problem?

**Comment by brendanjones** [comment]
Yep, still happens in the exact same way as I described in both comments above.

**Comment by nmat** [comment]
@brendanjones can you test rambox with a new user? That is to see if there is a problem with the data directory for rambox using a new user is easy on MacOS

**Comment by brendanjones** [comment]
Eli5? I actually don't see how to make a new user :/

**Comment by nmat** [comment]
Rambox stores data on disk, even cache files. So creating a new user on the machine and testing a _clean_ install using a different user will help to determine if there could be a file stored on disk that could cause the problem.

Its a bad way to test I agree but it helps to find what could be wrong.

So create a new MacOS user and change to that one. Then install rambox on that user and see if that has the same problem.

**Comment by brendanjones** [comment]
Oh right, I thought you meant a new _Rambox_ user :p Will do it soon, pretty busy atm.
